api/Window.json

```diff
@@ -3292,7 +3292,11 @@
             "safari": {
               "version_added": "1"
             },
-            "safari_ios": "mirror",
+            "safari_ios": {
+              "version_added": "1",
+              "partial_implementation": true,
+              "notes": "This method will not function if the target is unspecified or is set to <code>_blank</code>."
```

Comment:
https://github.com/mdn/browser-compat-data/issues/21661#issuecomment-2188828634 provides more context about why `_blank` may seemingly not work.

Reply:
That comment talks about the programmatic side of WebKit, which isn't quite as helpful for developers to know about as the note created in this PR!

Reply:
How about this:

```suggestion
              "notes": "This method will not function if the <code>target</code> parameter is unspecified or set to <code>_blank</code>."
```